### PR TITLE
CNS-779: Begin block store danger

### PR DESCRIPTION
**Pull request by shleikes**

### CODE REVIEW

File: x/projects/keeper/creation.go [hunk 1]
```diff
@@ -264,6 +265,12 @@ func (k Keeper) snapshotProject(ctx sdk.Context, projectID string, block uint64)
 			utils.Attribute{Key: "project", Value: projectID},
 			utils.Attribute{Key: "block", Value: ctx.BlockHeight()},
 		)
+
+		details := map[string]string{
+			"projectID": projectID,
+			"block":     strconv.FormatInt(int64(block), 10),
+		}
+		utils.LogLavaEvent(ctx, k.Logger(ctx), types.ProjectResetFailEventName, details, "reset projects failed: unable to find project plan")
```
Comment (Yaroms) [inline x/projects/keeper/creation.go:273]: fix error:
project plan -> project

File: x/projects/keeper/creation.go [hunk 2]
```diff
@@ -276,6 +283,12 @@ func (k Keeper) snapshotProject(ctx sdk.Context, projectID string, block uint64)
...
```
Comment (Yaroms) [inline x/projects/keeper/creation.go:291]: fix the error message here

File: x/projects/keeper/creation.go (hunk 1 above)
Comment (shleikes) [reply x/projects/keeper/creation.go]: Fixed in d30c3711ed7dbfef50f7cdb3f2b39d2d54695f42

File: x/projects/keeper/creation.go (hunk 2 above)
Comment (shleikes) [reply x/projects/keeper/creation.go]: Fixed in d30c3711ed7dbfef50f7cdb3f2b39d2d54695f42

File: x/projects/keeper/creation.go [hunk 3]
```diff
@@ -276,6 +283,12 @@ func (k Keeper) snapshotProject(ctx sdk.Context, projectID string, block uint64)
 			utils.Attribute{Key: "project", Value: projectID},
 			utils.Attribute{Key: "block", Value: ctx.BlockHeight()},
 		)
+
+		details := map[string]string{
+			"projectID": projectID,
+			"block":     strconv.FormatInt(int64(block), 10),
+		}
+		utils.LogLavaEvent(ctx, k.Logger(ctx), types.ProjectResetFailEventName, details, "reset projects failed: unable to find project")
```
[the remark this answers is not in the document]
Comment (shleikes) [reply x/projects/keeper/creation.go:291]: Fixed in 12cd63fd932f793de835ae768cdfe2037adce227

[the remark this answers is not in the document]
Comment (shleikes) [reply x/projects/keeper/creation.go]: Fixed in 12cd63fd932f793de835ae768cdfe2037adce227

**Review by Yaroms — APPROVED** [review]

**Pull request merged**